addons/norwegian_armed_forces/vehicles/index.hpp

```diff
@@ -65,8 +65,30 @@ class _xx_ACE_tourniquet { \
 class _xx_ACE_EarPlugs { \
 	name = "ACE_EarPlugs"; \
 	count = 5; \
+}; 
+
+#define VEHICLE_AT_AMMO \
+class _xx_MRAWS_HE_F { \
+	name = "MRAWS_HE_F"; \
+	count = 1; \
+}; \
+class _xx_MRAWS_HEAT_F { \
+	name = "MRAWS_HEAT_F"; \
+	count = 1; \
+};
+
+#define VEHICLE_AT_AMMO_EXTE \
```

Comment:
Can we have a name that makes more sense here? Is this intended for armoured or what?

Reply:
EXTE was put in as shorthand for extended. I can put it in as the full word though

Reply:
Aye, probably best